src/charactersheet/services/character/status_service_components/healthiness_component.js

```diff
@@ -53,13 +53,17 @@ export function HealthinessStatusServiceComponent() {
 
             if (deathSaves.length > 0) {
                 for(var i=0; i<3; i++) {
```

Comment:
Please make sure to have spaces in between `for` and the first parentheses. Make sure to check your other statements.